src/adapters/express.ts

```diff
@@ -7,6 +7,10 @@ export class ExpressConn<S> implements Conn<S> {
 
   constructor(readonly req: express.Request, readonly res: express.Response) {}
 
+  public clearCookie(name: string, options: CookieOptions) {
+    this.res.cookie(name, undefined, options)
```

Comment:
express has an explicit `clearCookie` too (http://expressjs.com/en/api.html#res.clearCookie), I think is cleaner